portfolio/src/main/webapp/index.html

```diff
@@ -19,17 +19,29 @@ <h1>Zhe Chen's Portfolio</h1>
       <button onclick="addFact()">This is Zhe!</button>
       <div id="fact-container"></div>
       <form action="/form-handler" method="POST">
-        <p>Enter some text here:</p>
-        <textarea name="text-input">hello world</textarea>
+        <p>Enter your favorite movie here:</p>
+        <textarea name="text-input">My favorite movie is </textarea>
         <br/>
         <input type="submit" />
       </form>
       <script src="script.js"></script>
+  </body>
+  <body onload="loadMovies()">
+    <h1>Get Movie List</h1>
+    <form method="POST" action="/list-movies">
+      <button>Get the list</button>
+    </form>
+    <hr/>
+    <ul id="movie-list"></ul>
+  </body>
```

Comment:
why are there 2 <body> tags? See https://www.w3schools.com/tags/tag_body.asp: "There can only be one <body> element in an HTML document."

Reply:
Change to one body only